bin/generate-docs.sh

```diff
@@ -85,11 +86,13 @@ function generate_scanner_docs() {
       "docs/README.DockerHub-Scanner.md" \
       "${scanner_dir}/.helm-docs.gotmpl" \
       "${HELM_DOCS_DIR}/README.DockerHub-Scanner.md.gotmpl"
+      # XXX: #754 Why is here no generation for ArtifactHub?
   else
     log "No scanner found at '${scanner_image_dir}'! Skipping scanner doc."
   fi
 
   log "Generating main doc..."
+  # XXX: #754 Why is here no generation for DockerHub?
```

Comment:
This is for the scanner related helm chart, the images for parser & custom scanner (if existient) where already published before, this now generates the readme for the helm chart.